aws/interface.go

```diff
@@ -107,6 +108,9 @@ func (c *interfaceClient) NewInterfaceOnSubnetAtIndex(index int, secGrps []strin
 		}
 		for _, intf := range newInterfaces {
 			if intf.Mac == *resp.NetworkInterface.MacAddress {
+				// New IP. Timestamp the addition as a free IP.
+				registry := &Registry{}
+				registry.TrackIP(net.ParseIP(*resp.NetworkInterface.PrivateIpAddress))
```

Comment:
error handling in case metadata gives us something not IP like? (we'll track nil and then boom)